### PR TITLE
conda-package-handling 2.0.2

### PR DESCRIPTION
**Pull request by dholth**
- uses [conda-package-streaming](https://github.com/AnacondaRecipes/conda-package-streaming-feedstock), ~~not yet Anaconda-ized.~~
- depends on python-zstandard (package name is just zstandard)
- faster

### CODE REVIEW

**Comment by dbast** [comment]
requires https://github.com/AnacondaRecipes/conda-package-streaming-feedstock/pull/1

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
   entry_points:
     - cph = conda_package_handling.cli:main
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - python                                 # [build_platform != target_platform]
```
Comment (katietz) [inline recipe/meta.yaml]: this stuff needs to be removed.  we do not have build_platform being defined ...  no cross compilation ...

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (dholth) [reply recipe/meta.yaml]: Will we ever be able to keep our recipes more similar to conda-forge, won't this conditional always be false anyway?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,34 +1,32 @@
...
+    - python >=3.7
     - pip
-    - cython
-    - _libarchive_static_for_cph ==3.6.1
-    - msinttypes  # [win and vc<14]
-    - setuptools
     - wheel
   run:
+    - python >=3.7
     - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Duplication of `python`. Should be a plain `python`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -40,33 +38,26 @@ test:
...
+    - pytest -v --cov=conda_package_handling --color=yes tests/
 
 about:
   home: https://github.com/conda/conda-package-handling
   dev_url: https://github.com/conda/conda-package-handling
-  doc_url: https://github.com/conda/conda-package-handling/blob/main/README.md
+  doc_url: https://conda.github.io/conda-package-handling/
   license: BSD-3-Clause
   license_family: BSD
   license_url: https://github.com/conda/conda-package-handling/blob/main/LICENSE
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `license_url` can be removed

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,34 +1,32 @@
...
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.7
     - pip
-    - cython
-    - _libarchive_static_for_cph ==3.6.1
-    - msinttypes  # [win and vc<14]
-    - setuptools
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:27]: We still use `setuptools` for building a package, see https://github.com/conda/conda-package-handling/blob/3c90a2e751a2babed46701bb9c972b7b3b474650/setup.py#L4

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,34 +1,32 @@
...
 
 source:
   url: https://github.com/conda/conda-package-handling/archive/{{ version }}.tar.gz
-  sha256: 81e0771b8b10ab5c20678a5da714e36d97e321acfc3e5b304ec2e650ce84734b
+  sha256: ec0a2c51f442d98b935ee2288beb1ec6c7e25020b5961f8d09f35738b276b532
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]: Missing `skip: True  # [py<37`], see https://github.com/conda/conda-package-handling/blob/3c90a2e751a2babed46701bb9c972b7b3b474650/setup.py#L25

File: recipe/meta.yaml [hunk 6]
```diff
@@ -1,34 +1,32 @@
...
     - cph = conda_package_handling.cli:main
 
 requirements:
   build:
-    - {{ compiler('c') }}
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.7
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: If we build an **arch-specific** package then in `host` should `python` **without** pinning as `conda-build` will pick up the latest version

**Comment by skupr-anaconda** [comment]
> requires [AnacondaRecipes/conda-package-streaming-feedstock#1](https://github.com/AnacondaRecipes/conda-package-streaming-feedstock/pull/1)

`conda-package-streaming 0.7.0` is available on defaults

File: recipe/meta.yaml (hunk 3 above)
Comment (dholth) [inline recipe/meta.yaml]:
```suggestion
```

**Review by dbast — APPROVED** [review]

**Pull request merged**